src/compiler/AST/Common/Type.elm

```diff
@@ -103,6 +105,29 @@ toString state type_ =
         Unit ->
             ( "Unit", state )
 
+        Tuple t1 t2 ->
+            let
+                ( t1String, state1 ) =
+                    toString state t1
+
+                ( t2String, state2 ) =
+                    toString state1 t2
+            in
+            ( "(" ++ t1String ++ "," ++ t2String ++ ")", state2 )
```

Comment:
From `elm repl` it seems like the official implementation adds spaces there to make it look like `( Int, String )`; similarly with the Tuple3